spec/datadog/ci/contrib/cucumber/formatter_spec.rb

```diff
@@ -1,12 +1,10 @@
 # typed: ignore
-require 'ddtrace/contrib/support/spec_helper'
+require 'datadog/ci/contrib/support/spec_helper'
```

Comment:
`datadog/ci/contrib/support/spec_helper` is all CI-App tests need now, it includes `ddtrace/contrib/support/spec_helper`.